primitives/types/hash_test.go

```diff
@@ -0,0 +1,131 @@
+package types
+
+import (
+	"bytes"
+	"testing"
+
+	sc "github.com/LimeChain/goscale"
+	"github.com/stretchr/testify/assert"
+)
+
+var (
+	hash256Bytes = []sc.U8{
+		0x37, 0x37, 0x37, 0x37, 0x37, 0x37, 0x37, 0x37, 0x37, 0x37,
+		0x37, 0x37, 0x37, 0x37, 0x37, 0x37, 0x37, 0x37, 0x37, 0x37,
+		0x37, 0x37, 0x37, 0x37, 0x37, 0x37, 0x37, 0x37, 0x37, 0x37,
+		0x37, 0x37,
+	}
```

Comment:
Can we have similar values like this in hex for consistency and better readibility?